foundations/javascript_basics/fundamentals-1.md

```diff
@@ -41,8 +41,17 @@ The simplest way to get started is to simply create an HTML file with the JavaSc
 
 Save and open this file up in a web browser (you can use Live Server to do this!) and then <span id="access-devTools-console">open up the browser's console by right-clicking on the blank webpage and selecting "Inspect" or "Inspect Element".  In the 'Developer Tools' pane find and select the 'Console' tab</span>, where you should see the output of our `console.log` statement.
 
+
 > <span id="console-log">`console.log()` is the command to print something to the developer console in your browser. You can use this to print the results from any of the following articles and exercises to the console.</span> We encourage you to code along with all of the examples in this and future lessons.
 
+Another way to include is through an external script. This is very similar to linking external CSS docs to your website. 
```

Comment:
```suggestion
Another way to include JavaScript in a webpage is through an external script. This is very similar to linking external CSS docs to your website. 
```
I suggest this change because it felt like what was being and included and where didn't seem totally clear.